front/app/components/ProjectCard/index.tsx

```diff
@@ -451,7 +437,7 @@ const ProjectCard = memo<InputProps>(
       const endAt = phase?.data.attributes.end_at;
 
       let countdown: JSX.Element | null = null;
-      const inputTerm = getInputTerm(phases?.data, phase?.data);
```

Comment:
The `phases` where a fallback for when `phase` was `undefined`. But this `inputTerm` is only necessary when `phase` is defined. So could be removed.